.github/workflows/docs-guide-test.yml

```diff
@@ -1,7 +1,5 @@
 name: Test Docs Guide
-on:
-  pull_request:
-    branches: [main]
+on: [push, pull_request]
```

Comment:
so this is now running the tests for all pushes and pull requests to any branch? I think I generally support that but I'm just wondering if there was a specific reason why we weren't doing it in the first place 🤔 was it putting a straing on CI resources or something? maybe @1ucian0 or @HuangJunye remembers?